python/tvm/relay/frontend/tflite.py

```diff
@@ -98,6 +98,7 @@ def __init__(self, model, subgraph, exp_tab):
             'SPACE_TO_BATCH_ND': self.convert_space_to_batch_nd,
             'PRELU': self.convert_prelu,
             'TRANSPOSE_CONV': self.convert_transpose_conv,
```

Comment:
Please change to `self.convert_detection_postprocess`! We should keep the same code style as other convert function jn the dictionary.